3.4/x86_alpine/start.sh

```diff
@@ -0,0 +1,49 @@
+#!bin/sh
+
+# /etc/nginx/sites-available/moodle
+NGINX_CONFIG="/etc/nginx/sites-available/moodle.conf"
+
+if [ ! -z "${MOODOLE_MAX_BODY_SIZE}" ]
+then
+  sed -i '/client_max_body_size/c\\tclient_max_body_size '$MOODOLE_MAX_BODY_SIZE';' $NGINX_CONFIG
+fi
+
+if [ ! -z "${MOODOLE_BODY_TIMEOUT}" ]
+then
+  sed -i '/client_body_timeout/c\\tclient_body_timeout '$MOODOLE_BODY_TIMEOUT';' $NGINX_CONFIG
+fi
+
+# /etc/php7/php-fpm.conf
+PHP_FM_CONFIG="/etc/php7/php-fpm.conf"
+echo "[www]" >> $PHP_FM_CONFIG
+
+if [ ! -z "${MOODOLE_DB_URL}" ]
+then
+  echo "env[MOODOLE_DB_URL] = '$MOODOLE_DB_URL'" >> $PHP_FM_CONFIG
+fi
+
+if [ ! -z "${MOODOLE_DB_NAME}" ]
+then
+  echo "env[MOODOLE_DB_NAME] = '$MOODOLE_DB_NAME'" >> $PHP_FM_CONFIG
+fi
+
+if [ ! -z "${MOODOLE_DB_USER}" ]
+then
+  echo "env[MOODOLE_DB_USER] = '$MOODOLE_DB_USER'" >> $PHP_FM_CONFIG
+fi
+
+if [ ! -z "${MOODOLE_DB_PASS}" ]
+then
+  echo "env[MOODOLE_DB_PASS] = '$MOODOLE_DB_PASS'" >> $PHP_FM_CONFIG
+fi
+
+if [ ! -z "${MOODOLE_DB_PORT}" ]
+then
+  echo "env[MOODOLE_DB_PORT] = '$MOODOLE_DB_PORT'" >> $PHP_FM_CONFIG
+fi
+
+/usr/sbin/php-fpm7
+
+/usr/sbin/nginx
+
+tail -f /var/log/nginx/access.log
```

Comment:
I think this log address is not correct, I didn't see any logs redirected here 😉

Reply:
But I see logs in /var/log/nginx/access.log in moodle container. I am confused
<img width="996" alt="screen shot 2018-01-18 at 12 25 14 pm" src="https://user-images.githubusercontent.com/14645773/35112168-3a8abbe4-fc4b-11e7-9431-5dac8252413e.png">

Reply:
have you tried `docker log -f 7814143b6f05`?